src/Files.App/Services/PreviewPopupProviders/SeerProProvider.cs

```diff
@@ -43,5 +52,64 @@ public async Task<bool> DetectAvailability()
 			var handle = User32.FindWindow("SeerWindowClass", null).DangerousGetHandle();
 			return handle != IntPtr.Zero && handle.ToInt64() != -1;
 		}
+
+		private bool? _IsTrackSelectionSettingEnabledCache;
+		private bool IsTrackSelectionSettingEnabled
+		{
+			get
+			{
+				if (_IsTrackSelectionSettingEnabledCache is null)
+					_IsTrackSelectionSettingEnabledCache = DetectTrackSelectionSetting().Result;
+
+				return _IsTrackSelectionSettingEnabledCache.Value;
+			}
+		}
+
+		private Task<bool> DetectTrackSelectionSetting()
+		{
+			bool trackSelectedFile = true;
+
+			var keyName = @"\HKEY_CURRENT_USER\Software\Corey\Seer";
```

Comment:
```suggestion
			var keyName = @"HKEY_CURRENT_USER\Software\Corey\Seer";
```
Fixing it this way will prevent crashes, but it will always track the file regardless of the settings.

Reply:
Do you have a link to the Win32 version? I can't seem to locate it.

Reply:
Is this what you mean?
http://1218.io/